patches/chrome-tools-build-win-create_installer_archive.py.patch

```diff
@@ -1,33 +1,50 @@
 diff --git a/chrome/tools/build/win/create_installer_archive.py b/chrome/tools/build/win/create_installer_archive.py
-index ffbf9ec0727b9a81eb4b8488ae67b797ec81a437..cdd0f6769f6629b855a56f267724673a79fec836 100755
+index ffbf9ec0727b9a81eb4b8488ae67b797ec81a437..444a2a1bf4776f7db7f209d8a7b7ab8d55b54bb7 100755
 --- a/chrome/tools/build/win/create_installer_archive.py
 +++ b/chrome/tools/build/win/create_installer_archive.py
-@@ -20,7 +20,7 @@ import os
- import shutil
- import subprocess
- import sys
--
+@@ -42,6 +42,7 @@ SETUP_PATCH_FILE_PREFIX = "setup_patch"
+ TEMP_ARCHIVE_DIR = "temp_installer_archive"
+ VERSION_FILE = "VERSION"
+ 
 +from create_installer_archive_helper import SignAndCopyPreSignedBinaries, BraveCopyAllFilesToStagingDir
  
- ARCHIVE_DIR = "installer_archive"
+ g_archive_inputs = []
  
-@@ -112,6 +112,7 @@ def CopyAllFilesToStagingDir(config, distribution, staging_dir, build_dir,
+@@ -112,6 +113,7 @@ def CopyAllFilesToStagingDir(config, distribution, staging_dir, build_dir,
    if enable_hidpi == '1':
      CopySectionFilesToStagingDir(config, 'HIDPI', staging_dir, build_dir,
                                   verbose)
 +  BraveCopyAllFilesToStagingDir(config, staging_dir, g_archive_inputs)
  
    if include_snapshotblob == '1':
      CopySectionFilesToStagingDir(config, 'SNAPSHOTBLOB', staging_dir, build_dir,
-@@ -551,6 +552,7 @@ def main(options):
+@@ -155,7 +157,7 @@ def CopySectionFilesToStagingDir(config, section, staging_dir, src_dir,
+ 
+ def GenerateDiffPatch(options, orig_file, new_file, patch_file):
+   if (options.diff_algorithm == "COURGETTE"):
+-    exe_file = os.path.join(options.last_chrome_installer, COURGETTE_EXEC)
++    exe_file = os.path.join(options.build_dir, COURGETTE_EXEC)
```

Comment:
last_chrome_installer is directory that contains previous version's installer dir.
courgettes exe is generated in ./out.

Reply:
Or we can avoid this patch if we copy `courgette.exe` from `out/Release` to folder indicated by `last_chrome_installer` before running `build_delta_installer`. cc @mihaiplesa

Reply:
This change is deleted.
`courgette.exe` will be copied from out/Release to `last_chrome_installer` by pipeline.